operator/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 quarkus.operator-sdk.crd.apply=true
-quarkus.container-image.builder=jib
+quarkus.container-image.builder=docker
 quarkus.operator-sdk.crd.validate=false
 
 # Operator config
```

Comment:
I think we can add something like:
```
quarkus.docker.dockerfile-jvm-path=/Dockerfile
```
So that we could keep the Dockerfile in root of the module. See also: https://quarkus.io/guides/container-image#DockerOptions

Reply:
I tried that, but the existence check for `src/main/docker` seems to be a separate to the Dockerfile path one. They might just have that option in case you want a different filename for your Dockerfile?

The code is [here](https://github.com/quarkusio/quarkus/blob/3414e59da7d4340b98c0d85aa07637cefe88e520/extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerProcessor.java#L434), I checked it to see if there was any way to bypass the error it throws, but it doesn't appear so.